src/styles/blocks/stars.scss

```
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+
+  &__star {
+    @include star-size($star-width, $star-height);
+    margin-left: 4px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+
+  @for $starCount from 1 through 5 {
+    &--#{$starCount} .stars__star:nth-child(-n + #{$starCount}) {
```

remember about nesting
```suggestion
    &--#{$starCount} &__star:nth-child(-n + #{$starCount}) {
```